test/main.js

```diff
@@ -395,7 +408,11 @@ const runMockServerAndTests = () =>
     findMockServerId(runTests)
   })
 
-runMockServerAndTests()
-
+if (process.env.MOCK_SERVER === 'false') {
+  console.log('Tests will run without the use of the mock server')
```

Comment:
Better to explicitly state that the tests will be running against production to be clear

Reply:
Noted, I'll make that change.